conformance_tests/sysman/test_sysman_fabric/src/test_sysman_fabric.cpp

```diff
@@ -289,6 +289,127 @@ TEST_F(
   }
 }
 
+TEST_F(
+    FABRICPORT_TEST,
+    GivenValidFabricPortHandleWhenRetrievingMultiPortThroughputsThenValidThroughputAreReturned) {
+  for (auto device : devices) {
+    uint32_t count = 0;
+    auto fabric_port_handles = lzt::get_fabric_port_handles(device, count);
+    if (count == 0) {
+      FAIL() << "No handles found: "
+             << _ze_result_t(ZE_RESULT_ERROR_UNSUPPORTED_FEATURE);
+    }
+
+    auto multiport_throughPuts = lzt::get_multiport_throughputs(
+        device, count, fabric_port_handles.data());
+    ASSERT_EQ(fabric_port_handles.size(), count);
+
+    for (auto fabric_port_throughput : multiport_throughPuts) {
```

Comment:
Please do some fabric transfers, to measure valid througputs

Reply:
@joshuaranjan  Probably it needs some special set up?

Reply:
Yes Any setup with Xelink connection should be fine.
We can refer https://github.com/intel-innersource/libraries.compute.oneapi.level-zero.tests/blob/bce65380a48d95386af11b82d99a3bcf96f8104b/conformance_tests/core/test_fabric/src/test_fabric.cpp#L582C13-L582C38 I think